src/test/java/io/nats/client/support/ValidatorTests.java

```diff
@@ -545,55 +545,55 @@ public void testSemver() {
     }
 
     @Test
-    public void testListsAreEqual() {
+    public void testConsumerFilterSubjectsAreEquivalent() {
```

Comment:
changed the name to match the method being tested, which changed it's name to clearer.